commands/backuplxc.go

```diff
@@ -118,6 +107,27 @@ func (c *backupLXCCommand) Run(ctx *cmd.Context) error {
 	return group.Wait()
 }
 
+func getLXCContainerList(c *baseClientCommand) ([]lxcContainer, error) {
+	// Get a listing of all of the LXC containers in the environment.
+	var buf bytes.Buffer
+	rc, err := runViaSSH(
+		c.address,
+		c.getRemoteCommand(c.remoteCommand),
+		withStdout(&buf),
+	)
+	if err != nil {
+		return nil, errors.Annotatef(err, "running %s via SSH", c.remoteCommand)
+	}
+	if rc != 0 {
+		return nil, &cmd.RcPassthroughError{rc}
```

Comment:
I was mainly asking because I've seen it elsewhere in the code (and I may have cargo-culted it) but I'm not sure what it's for.